### ISSUE TITLE
Swift Metrics is included in the Package but is never used

### ISSUE THREAD

**Issue by adam-fowler** [swift-server/swift-service-lifecycle#78]
Is there a plan to use it at some point?

**Comment by ktoso** [comment]
I don't know what the intent here was @tomerd.
I guess one can do metrics about started/stopped lifecycle members but that's about it -- not super sure if that's very useful metrics tbh.

Removing might break builds for folks so we shouldn't just do that in minor releases though.

**Comment by adam-fowler** [comment]
@ktoso no real issue. It's just a little wasteful. And yes I can't see any real advantage to using metrics with this package. Unless I'm missing something.

**Comment by ktoso** [comment]
Yeah, agreed. I guess we can schedule for removing it when dropping a 2.x here. I'll let Tom chime in if he had specific use cases in mind here though first.

**Comment by ktoso** [comment]
Now that I wrote this... I guess timers for how long tasks are starting/stopping could be fun... 🤔

**Comment by adam-fowler** [comment]
Given a full 1.0 release hasn't been released, if it isnt required could it be removed for 1.0

**Comment by tomerd** [comment]
the original idea was to use it to report a start-up counter - in  past life we used such counter in production system as an indication of a service that is failing to start and gets rebooted again and again by the orchestration system (e.g. k8) - basically if we saw this counter spike it was a good proxy that something bad is going on and we should get an alert, etc.  maybe there are better way to do this nowadays but it was fairly useful.

durations for start-up and shut-down should also be fun!
